src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+   >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+   >
+  </head>
+  <body class="home-page__content">
+    <article class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      <h3 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+      <span class="card__term">Код товара: 195434</span>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__testimonials">Отзывов: 5</span>
+      </div>
+      <div class="card__price card-price">
```

Comment:
```suggestion
      <div class="card__price price">
```
It would be better to use such namings